R/model.R

```diff
@@ -86,11 +86,11 @@ get_model <- function(workspace,
 register_model <- function(workspace,
                            model_path,
                            model_name,
+                           datasets,
```

Comment:
this should be an optional parameter, right?

Reply:
Good catch - thanks.